scss/1-settings/_config.scss

```diff
@@ -0,0 +1,5 @@
+// ============================================================================
+// Config flags
+// ============================================================================
+
+$cads-enable-external-link-icon: true !default;
```

Comment:
should be `$cads-enable-opens-in-new-tab-icon`

LOL JK

Reply:
👏